datascience/tables.py

```diff
@@ -5678,12 +5678,46 @@ def asdict(self):
             return collections.OrderedDict(zip(self._table.labels, self))
 
     class Rows(collections.abc.Sequence):
-        """An iterable view over the rows in a table."""
+        """
+        An iterable view over the rows in a table.
+
+        args:
+        table- accepts a table object of class Table()
```

Comment:
Can we please follow the standard formatting for args/return values? Here's a good example: https://github.com/data-8/datascience/blob/master/datascience/tables.py#L380-L402

E.g.:
```
...

Args:
    table (str): ...
```

Reply:
Also "Table" here can be written as Table or `Table` - but no parenthesis please.